molecule/cluster_cookie/prepare.yml

```diff
@@ -0,0 +1,22 @@
+---
+
+- name: 'Configure instances'
+  hosts: cluster
+  roles:
+    - ansible-cartridge
+  become: true
+  become_user: root
+  gather_facts: false
+  vars:
+    cartridge_scenario:
```

Comment:
Is it impossible to use other steps?
If it's impossible, let's add to doc that without cookie impossible to use ... steps

Reply:
It's possible, I've just removed unnecessary configuration to reduce test duration.